content/en/developers/metrics/_index.md

```diff
@@ -17,20 +18,28 @@ further_reading:
+
+**Note**: Administrative users (those with [Datadog Admin roles][6] can see the total custom metrics per hour and the top 500 custom metrics for their account in the [usage details page][7]. Learn more about [how Custom metrics are counted][5].
+
+## Metric definition
+
+A Datadog metric is defined by the properties below. Refer to the [Metrics Introductionumentation][8] to learn how to graph metrics within Datadog.
 
 | Property         | Description                                                                                                                                               |
-|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `<METRIC_NAME>`  | The [name of your metric](#naming-metrics).                                                                                                               |
 | `<METRIC_VALUE>` | The value of your metric.                                                                                                                                 |
-| `<TIMESTAMP>`     | The timestamp associated with the metric value. **Note**: Metric timestamps cannot be more than 10 minutes in the future or more than 1 hour in the past. |
+| `<TIMESTAMP>`    | The timestamp associated with the metric value. **Note**: Metric timestamps cannot be more than 10 minutes in the future or more than 1 hour in the past. |
 | `<CONTEXT>`      | The set of tags associated with your metric.                                                                                                              |
-| `<METRIC_TYPE>`  | The type of your metric. See the [metric type documentation][2].                                                                                          |
+| `<METRIC_TYPE>`  | The type of your metric. See the [metric type documentation][9].                                                                                          |
 | `<INTERVAL>`     | If the `<TYPE>` of the metric is RATE or COUNT, it defines the corresponding interval.                                                                    |
 
-If a metric is not submitted from one of the [350+ Datadog integrations][3], it's considered a [Custom Metric][4]. **Note**: Some standard integrations [emit custom metrics][5].
-
-### Naming metrics
+### Naming custom metrics
 
 Follow these rules for naming metrics:
```

Comment:
Again I think this section is unnecessary and is directly copied over from: https://docs.datadoghq.com/developers/metrics/. If anything let's just link back to that page

Reply:
indeed that's the same page.